jbi/router.py

```diff
@@ -75,6 +76,20 @@ async def bugzilla_webhook(
     return await execute_or_queue(webhook_request, queue, actions)
 
 
+@router.get(
+    "/dl_queue/",
```

Comment:
Just thinking out loud here, but maybe this `/dl_queue/` route could list all of the items (by identifier only), then `/dl_queue/<identifier>/` could be the detail view of one bug? And/or `/dl_queue/<bug_id>/<identifier>/`

Reply:
I would vote to start with something simple like this and iterate if we face loading issues. This API endpoint is not meant to be used publicly anyway. We could also turn it to HTML in case we want to prevent consumers to rely on its format